src/ErrorElement.tsx

```diff
@@ -9,11 +9,7 @@ import { css } from '@emotion/react'
 
 import GrotCrashed from '@/assets/grot-crashed.svg'
 import { ArrowLeftIcon } from '@radix-ui/react-icons'
-
-const handleCreateIssue = () =>
-  window.studio.browser.openExternalLink(
-    'https://github.com/grafana/k6-studio/issues'
-  )
+import { ExternalLink } from './components/ExternalLink'
```

Comment:
I think we should also add an icon to make it a bit clearer that the link will open in the browser, something like the [external link icon in Grafana UI](https://developers.grafana.com/ui/latest/index.html?path=/story/general-textlink--example)

Reply:
That would be a great addition!